packages/docs/src/routes/docs/components/events/index.mdx

```diff
@@ -188,11 +211,18 @@ Notice that `on:click` attribute contains three pieces of information:
 2. `Counter_button_onClick`: The symbol which needs to be retrieved from the lazy-loaded chunk.
 3. `[0]`: An array of lexically captured variable references (State of the closure).
 
-In our case `() => store.count++` only captures `store`, and hence it contains only a single reference `0`. `0` is an index into the `q:obj` attribute that contains a reference to the actual serialized object, referring to `store`. (The exact mechanisms and syntax is an implementation detail that can change at any time)
+In our case `() => store.count++` only captures `store`, and hence it contains only a single reference `0`. 
+
+`0` is an index into the `q:obj` attribute that contains a reference to the actual serialized object, referring to `store` (The exact mechanisms and syntax is an implementation detail that can change at any time).
```

Comment:
this is actually super old, outdated and too implementation detail, we should remove this, but i am ok to leave it for another PR

Reply:
Fine with removing it @manucorporat, but which part?